kong/tools/utils.lua

```diff
@@ -473,16 +473,17 @@ end
 -- @param t2 The second table
 -- @return The (new) merged table
 function _M.table_merge(t1, t2)
-  if not t1 then
-    t1 = {}
+  local res = {}
```

Comment:
I think if we're going to update this code we might as well share it with the PDK so that there's only one implementation. `kong/pdk/table.lua` can just contain `merge = require("kong.tools.utils").table_merge` in its code.

Reply:
+1